service/raft/raft_group0.cc

```diff
@@ -1495,7 +1511,8 @@ future<> raft_group0::upgrade_to_group0(service::storage_service& ss, cql3::quer
         try {
             co_await self.do_upgrade_to_group0(start_state, ss, qp, mm, cdc_gen_service);
             co_await self._client.set_group0_upgrade_state(group0_upgrade_state::use_post_raft_procedures);
-            upgrade_log.info("Raft upgrade finished.");
+            upgrade_log.info("Raft upgrade finished. Disabling migration_manager schema pulls.");
+            co_await mm.disable_schema_pulls();
```

Comment:
Note that during Raft upgrade procedure, there will be a window of time when schema is pulled both by `migration_manager` and new schema changes are pushed through Raft - until all nodes reach this call.

@tgrabiec